cape-char.el

```diff
@@ -25,44 +25,35 @@
 ;;; Code:
 
 (require 'cape)
+(require 'quail)
 
 (autoload 'thing-at-point-looking-at "thingatpt")
 
 ;; Declare as pure function which is evaluated at compile time. We don't use a
 ;; macro for this computation since packages like `helpful' will
 ;; `macroexpand-all' the expensive `cape-char--define' macro calls.
 (eval-when-compile
```

Comment:
> We shouldn't require quail here, such that we don't load it at runtime. Instead the require should go to the cape-char--translation function.

Would it be better to put it outside the translation function, but inside `eval-when-compile` so it gets loaded once, and not loaded each time the translation function is called?
```suggestion
(eval-when-compile
  (require 'quail)
```

Reply:
No, I would remove the eval-when-compile and also call the function directly from the macro. We can do that given that your approach is cleaner and we also have to do that if we want to make the define function public.

Reply:
Ah I see, thanks, I'll include the change as you suggested originally in the next commit.